File: Parte 1/PDFGenerationServer/PDFGenerationServer/Models/DTO/LogMessageDTO.cs
namespace PDFGenerationServer.Models.DTO
{
    public class LogMessageDTO
    {
        public string CorrelationId { get; set; }
        public string Service { get; set; }
        public string Endpoint { get; set; }
        public string TimeStrap { get; set; } // aqui
        public object Payload { get; set; }   // y aqui, cambie el nombre porque estaba mal escrito xd
        public bool Success { get; set; }
    }
}

File: Parte 3/Messages Server/src/models/LogMessageDTO.js
class LogMessageDTO {
  constructor({
    correlationId,
    service,
    endpoint,
    timeStrap = new Date().toISOString(),
    payload = {},
    success = false
  }) {
    this.correlationId = correlationId;
    this.service = service;
    this.endpoint = endpoint;
    this.timeStrap = timeStrap; 
    this.payload = payload;     
    this.success = success;
  }
}

module.exports = LogMessageDTO;
